src/lib/bit/bit.h

```diff
@@ -303,6 +303,225 @@ bit_set_range(void *data, size_t pos, size_t count, bool val)
 	 */
 }
```

Comment:
Nit. I guess we don't need to mention bugfix here, only in EE.

Reply:
Done.